hack/make/.binary

```diff
@@ -78,6 +78,9 @@ source "${MAKEDIR}/.go-autogen"
 		GCFLAGS="all=-N -l"
 	fi
 
+	CGO_LDFLAGS=-latomic
+	export CGO_LDFLAGS
```

Comment:
If this works, it may be worth factoring it out into a .go file so that it can be build-tagged to ARMv5, and so that plain old `go build` would also work without any special environment variables.

```go
//go:build arm.5

// #cgo LDFLAGS: -latomic
import "C"
```

Reply:
Yeah, the amount of options we need is growing rapidly 😔

This was indeed just a quick push to see if it worked (almost forgot to add it again 😅)